internal/xds/translator/translator.go

```diff
@@ -1032,30 +1033,22 @@ const (
 	EDS
 )
 
-// defaultCertificateName is the default location of the system trust store, initialized at runtime once.
-//
-// This assumes the Envoy running in a very specific environment. For example, the default location of the system
-// trust store on Debian derivatives like the envoy-proxy image being used by the infrastructure controller.
-//
-// TODO: this might be configurable by an env var or EnvoyGateway configuration.
-var defaultCertificateName = func() string {
-	switch runtime.GOOS {
-	case "darwin":
-		// TODO: maybe automatically get the keychain cert? That might be macOS version dependent.
-		// For now, we'll just use the root cert installed by Homebrew: brew install ca-certificates.
-		//
-		// See:
-		// * https://apple.stackexchange.com/questions/226375/where-are-the-root-cas-stored-on-os-x
-		// * https://superuser.com/questions/992167/where-are-digital-certificates-physically-stored-on-a-mac-os-x-machine
-		return "/opt/homebrew/etc/ca-certificates/cert.pem"
-	default:
-		// This is the default location for the system trust store
-		// on Debian derivatives like the envoy-proxy image being used by the infrastructure
-		// controller.
-		// See https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/security/ssl
-		return "/etc/ssl/certs/ca-certificates.crt"
-	}
-}()
+var certificatePath = cmp.Or(
```

Comment:
we have gotten away w/o introducing any ENV VARs or annotations in this project, hope we can continue with that streak.
any configurations needed, are done by the APIs, for e.g. for this case it could live under EnvoyGateway
https://gateway.envoyproxy.io/docs/api/extension_types/#envoygatewayhostinfrastructureprovider

Reply:
good to know